skins/oasis/js/Tracking.js

```diff
@@ -598,13 +645,13 @@ jQuery(function ($) {
 
 	/** wiki-nav **/
 
-	$wikiHeader.on('mousedown', 'a', function (e) {
+	$wikiHeader.on('mousedown', 'a', function (event) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)